HoloLensCommander/HoloLensCommander/Dialogs/SetIpdDialog.xaml.cs

```diff
@@ -18,12 +18,17 @@ public sealed partial class SetIpdDialog : ContentDialog
         /// <summary>
         /// Initializes a new instance of the <see cref="SetIpdDialog" /> class.
         /// </summary>
-        /// <param name="userInfo">TagInformation object containing information associated with the HoloLens.</param>
-        public SetIpdDialog(UserInformation userInfo)
+        /// <param name="address">The address of the HoloLens.</param>
+        /// <param name="userInfo">UserInformation object containing information associated with the HoloLens.</param>
+        public SetIpdDialog(
+            string address,
+            UserInformation userInfo)
```

Comment:
no, the address comes from the calling monitor. I pass it as a string in the spirit of less is more :)